### PR TITLE
[ENG-895] Allow setting var from standard input

### PR DESCRIPTION
**Pull request by tebeka**

### CODE REVIEW

File: cmd/ak/cmd/vars/set.go [hunk 1]
```diff
@@ -28,7 +31,20 @@ var setCmd = common.StandardCommand(&cobra.Command{
 			return fmt.Errorf("invalid variable name %q: %w", args[0], err)
 		}
 
-		ev := sdktypes.NewVar(n, args[1], secret).WithScopeID(id)
+		var value string
```
Comment (itayd) [inline cmd/ak/cmd/vars/set.go:34]: i think better to have a flag to read from stdin, no?

Comment (daabr) [reply cmd/ak/cmd/vars/set.go:34]: e.g. https://github.com/autokitteh/autokitteh/blob/main/cmd/ak/cmd/manifest/validate.go

Comment (tebeka) [reply cmd/ak/cmd/vars/set.go:34]: > i think better to have a flag to read from stdin, no?

The common Unix practice is to read from stdin (`grep`, `sed` ...) if no file is specified. Or maybe use `-` (see [here](https://docs.python.org/3/library/argparse.html#argparse.FileType) for example) as the file name. I prefer the former.

There also what `curl` does which is when value is prefixed with `@` it's a name of the file to read from (see `Variables` section [here](https://curl.se/docs/manpage.html)), but it's less flexible than standard output that can also accept a pipe ...

@daabr Your example is like the current behaviour. Am I missing something?

**Review by ig-ra — APPROVED** [review]

File: cmd/ak/cmd/vars/set.go [hunk 2]
```diff
@@ -28,7 +31,20 @@ var setCmd = common.StandardCommand(&cobra.Command{
...
+		if len(args) == 1 {
```
Comment (daabr) [inline cmd/ak/cmd/vars/set.go]: Optional nit: given that `== 2` is the "trivial" case, and it takes only 1 line of code, it's easier to read the code if you reverse the order of the branches.

File: cmd/ak/cmd/vars/set.go (hunk 1 above)
Comment (daabr) [reply cmd/ak/cmd/vars/set.go:34]: Sorry for not being clear, my comment was just intended to provide a concrete example ("e.g.") for how we currently handle files vs stdin.

Personally, I prefer the practice of indicating the usage of stdin with  `-`, but that's not what we usually do in AK.

I'm ok with submitting your PR, but we should use a single convention in all relevant commands.

**Review by daabr — APPROVED** [review]

File: cmd/ak/cmd/vars/set.go (hunk 1 above)
Comment (daabr) [reply cmd/ak/cmd/vars/set.go:34]: I opened https://linear.app/autokitteh/issue/ENG-928/cli-standardization-using-stdin-instead-of-filename for follow-up

Comment (daabr) [reply cmd/ak/cmd/vars/set.go:34]: I opened https://linear.app/autokitteh/issue/ENG-928/cli-standardization-using-stdin-instead-of-filename for follow-up

File: cmd/ak/cmd/vars/set.go (hunk 2 above)
Comment (tebeka) [reply cmd/ak/cmd/vars/set.go]: Done.

**Pull request merged**